packages/offix-datastore/src/storage/adapters/IndexedDBStorageAdapter.ts

```diff
@@ -56,6 +56,8 @@ export class IndexedDBStorageAdapter implements StorageAdapter {
 
             this.stores.forEach(({ name, keyPath }) => {
                 if (existingStoreNames.contains(name)) { return; }
+                logger("Creating store", name, keyPath);
+                // TODO hardcoded id and unused keypath
```

Comment:
Huge problem we have now is that we assume that id will be and `id` which is not going to work even by default as we move now to `_id` for Mongo in coming weeks. 

CC @ssd72